### PR TITLE
Created skeleton code for BankWithUs

### PR DESCRIPTION
**Pull request by tyuyang**
Code is based off [this site](https://nus-cs2113-ay2223s2.github.io/website/schedule/week7/project.html).

### CODE REVIEW

File: src/main/java/seedu/bankwithus/BankWithUs.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package seedu.bankwithus;
+
+public class BankWithUs {
+
+    private Storage storage;
+    private Ui ui;
+    private AccountList accounts;
+
+    public static final String FILE_PATH = "data/save.txt";
```
Comment (vishnuvk47) [inline src/main/java/seedu/bankwithus/BankWithUs.java:9]: Fails a CI build task with Error: Static variable definition in wrong order. [DeclarationOrder]
declare variables in order to resolve check style violation:
1. Constants (public static final)
2. Static variables (public static)
2. Instance variables (private)

**Review by vishnuvk47 — COMMENTED** [review]
check style violation at BankWithUS.java line 9.

reorder declarations as such:
Constants (public static final)
Static variables (public static)
Instance variables (private)

**Comment by tyuyang** [comment]
Done modifying BWU.java to comply with checkstyle, should be ready to merge now

**Comment by tyuyang** [comment]
Merged since @vishnuvk47 has already looked at it. Mostly cos I wanted to make another pull request.

**Pull request merged**